tmpl/blog/mirage-entropy.md

```diff
@@ -0,0 +1,362 @@
+# Organized chaos: managing randomness
+
+This post gives a bit of background on the Random Number Generator (RNG) in the
+new MirageOS 2.5.0.
+
+First we give background about why RNGs are really critical for security, then
+we try to clarify the often-confused concepts of "randomness" and "entropy" as
+used in this context, and finally we explore the challenges of harvesting
+good-quality entropy in a unikernel environment.
+
+## The race to... wherever
+
+Security software needs to play dice.
+
```

Comment:
Suggest replacing "The race to... wherever" with "Security software must play dice" as a heading.